PopPUNK/assign.py

```diff
@@ -119,7 +125,12 @@ def assign_query(dbFuncs,
             for reference in refFile:
                 rNames.append(reference.rstrip())
     else:
-        rNames = getSeqsInDb(ref_db + "/" + os.path.basename(ref_db) + ".h5")
+        if os.path.isfile(distances + ".pkl"):
+            rNames = readPickle(distances, enforce_self = True, distances=False)[0]
+        elif update_db:
+            sys.stderr.write("Reference distances missing, cannot use --update-db\n")
```

Comment:
```suggestion
            sys.stderr.write("Reference distances missing, cannot use --update-db\n")
            sys.exit(1)
```

I forgot to add the exit here